devicepluginunipi.cpp

```diff
@@ -427,22 +427,27 @@ void DevicePluginUniPi::setupDevice(DeviceSetupInfo *info)
         if (!neuronExtensionInterfaceInit())
             return info->finish(Device::DeviceErrorSetupFailed, QT_TR_NOOP("Error setting up Neuron."));
 
-        int slaveAddress = device->paramValue(neuronXS10DeviceSlaveAddressParamTypeId).toInt();
-        NeuronExtension *neuronExtension = new NeuronExtension(NeuronExtension::ExtensionTypes::xS10, m_modbusRTUMaster, slaveAddress, this);
+        int slaveAddress;
+        NeuronExtension *neuronExtension;
```

Comment:
may be uninitialized and even will crash later on if by some reason the `else` is hot in the next `if`